Library/Homebrew/api/formula.rb

```diff
@@ -101,6 +102,71 @@ def write_names_and_aliases(regenerate: false)
             end
           end
         end
+
+        REQUIRED_FIELDS = %w[
+          aliases
+          bottle
+          build_dependencies
+          caveats
+          conflicts_with
+          conflicts_with_reasons
+          dependencies
+          deprecation_date
+          deprecation_reason
+          desc
+          disable_date
+          disable_reason
+          full_name
+          homepage
+          keg_only_reason
+          license
+          link_overwrite
+          name
+          oldnames
+          optional_dependencies
+          options
+          post_install_defined
+          pour_bottle_only_if
+          recommended_dependencies
+          requirements
+          revision
+          ruby_source_checksum
+          ruby_source_path
+          service
+          tap
+          tap_git_head
+          test_dependencies
+          urls
+          uses_from_macos
+          uses_from_macos_bounds
+          variations
+          version_scheme
+          versioned_formulae
+          versions
+        ].freeze
+        private_constant :REQUIRED_FIELDS
+
+        sig { params(formula_hash: Hash).returns(Hash) }
+        def slim_hash(formula_hash)
```

Comment:
Again: don't need this yet but probably want to have tests ensuring that formulae are loaded as expected from this JSON.